generator/toyota/toyota_nodsu_hybrid_pt.dbc

```diff
@@ -2,12 +2,6 @@ CM_ "IMPORT _toyota_2017.dbc";
 CM_ "IMPORT _comma.dbc";
 CM_ "IMPORT _toyota_nodsu_common.dbc";
 
-BO_ 295 GEAR_PACKET: 8 XXX
```

Comment:
no reason to delete this one. let's just call it something else

Reply:
Do we still want to keep this one?

Reply:
yeah, call it something like GEAR_PACKET_HYBRID